src/main/scala/mesosphere/marathon/api/v2/AppsResource.scala

```diff
@@ -284,11 +284,16 @@ class AppsResource @Inject() (
     }
   }
 
-  private def enrichedTasks(app: AppDefinition): Seq[EnrichedTask] =
-    taskTracker.get(app.id).map { task =>
-      val hcResults = result(healthCheckManager.status(app.id, task.getId))
-      EnrichedTask(app.id, task, hcResults)
-    }.to[Seq]
+  private def enrichedTasks(app: AppDefinition): Seq[EnrichedTask] = {
+    val tasks = taskTracker.get(app.id).map { task =>
```

Comment:
Sure, if you know the return type of `taskTracker.get`.  Anyhow, it's just a suggestion ;)